ntbs-service/wwwroot/css/header.scss

```diff
@@ -54,7 +65,7 @@
     display: none;
 }
 
-@media (min-width: 48.0625em) {
+@media screen and (min-width: 48.0625em) {

```

Comment:
I don't think I mind too much but the use of @media print and @media screen for both adding and removing styles in each case seems a bit confusing - perhaps we could keep @media print for when we want to alter styling for printing and @media screen to apply styling that we don't want in print? or just keep to one or the other (might be easier to chat about this offline as it's quite a vague comment!)